README.md

```diff
@@ -1,73 +1,44 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Leaderboard App
 
+This project send data to the leaderboad api and get it from the same api to display the score when the person adds it
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JavaScript
 
-## Live Demo (if available)
+## Live Demo
 
 [Live Demo Link](https://livedemo.com)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+Github profile and code editor
 
 ### Setup
+copy the repository from the drop down code menu
 
 ### Install
+clone the repository to your local machine through your code editor using git clone
```

Comment:
- [ ] Kindly ensure to add proper/ detailed instructions on how to **setup** and  **run** your application, so as to make your ReadMe appear more professional